examples/fscloud/README.md

```diff
@@ -1,19 +1,21 @@
-# Financial Services Cloud profile example
+# Financial Services compliant example
 
-An end-to-end example that uses the [Profile for IBM Cloud Framework for Financial Services](../../modules/fscloud/) to deploy an instance of IBM Cloud Databases for Redis.
+This example uses the [Profile for IBM Cloud Framework for Financial Services](../../modules/fscloud/) to provision an IBM Cloud Databases for Redis instance.
```

Comment:
you will need to update the hyperlink to full url (aka `https://github.com/terraform-ibm-modules/terraform-ibm-icd-redis/tree/main/modules/fscloud`) for it to link in registry